AirCasting/Utils/Strings.swift

```diff
@@ -1052,4 +1052,8 @@ struct Strings {
     enum ProtectedScreen {
         static let title: String = NSLocalizedString("Please, do not force close the app while recording a session!", comment: "")
     }
+    
+    enum SearchFollowCompleteScreen {
+        static let missingStreams: String = NSLocalizedString("Some streams are missing, although you can still follow this session.", comment: "")
```

Comment:
I think if there are less than 5 streams for airbeam sessions it means there was some error on the way and I'm not sure we should allow the user to add an incomplete session. But I'm leaving it for you to decide.